OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -227,19 +160,22 @@ static object LoadBuildingCategories(MiniYaml yaml)
 		public object Create(ActorInitializer init) { return new HackyAI(this, init); }
 	}
 
-	public sealed class HackyAI : ITick, IBot, INotifyDamage
+	public sealed class HackyAI : ITick, IBot, INotifyDamage, IBotPositionsUpdated
 	{
 		// DEPRECATED: Modules should use World.LocalRandom.
 		public MersenneTwister Random { get; private set; }
 		public readonly HackyAIInfo Info;
 
-		public CPos GetRandomBaseCenter()
+		public CPos GetRandomBaseCenter
```

Comment:
Style nit: this would be best kept as a method. It's doing a calculation and returning a different result each time, whereas properties are generally used for state-like data.